### ISSUE TITLE
Where to define routes

### ISSUE THREAD

**Issue by Geokoumpa** [karafka/karafka#58]
Hello,

I was not able to use routing because I get the error 

`undefined method 'routes' for App:Class (NoMethodError)`

Where are routes supposed to be defined? I added mine inside app.rb, under App class definition.

Any ideas about what I might be doing wrong?

Also I had to use Karafka::BaseController insead of ApplicationController because the latter was undefined. I am using v 0.4.

thanks

**Comment by mensfeld** [comment]
@Geokoumpa  here you go:

```ruby
ENV['RACK_ENV'] ||= 'development'
ENV['KARAFKA_ENV'] ||= ENV['RACK_ENV']

Bundler.require(:default, ENV['KARAFKA_ENV'])

# App class
class App < Karafka::App
  setup do |config|
    config.kafka_hosts = %w( 127.0.0.1:9092 )
    config.zookeeper_hosts = %w( 127.0.0.1:2181 )
    config.worker_timeout = 60 # 1 minute,
    config.wait_timeout = 60 # 1 minute
    config.max_concurrency = 5
    config.name = 'example_app'
    config.redis = {
      url: 'redis://localhost:6379'
    }
  end
end

Karafka::Loader.new.load(App.root)

# Routes need to be drawn after the application code is loaded because routes rely on your code
# Please read documentation for more details on how to draw routes
App.routes.draw do
  # topic :example do
  #   controller ExampleController
  #   interchanger CustomInterchanger
  # end
end
```

**Comment by mensfeld** [comment]
@Geokoumpa please close this issue if it helped

**Comment by Geokoumpa** [comment]
I just did a bundle update. Although I had just installed karafka this morning, it made a difference.

What about ApplicationController? Why is it undefined?

> Controllers should inherit from ApplicationController

**Comment by mensfeld** [comment]
@Geokoumpa that is because yesterday we released a 0.4 that is huuuuuuuuuge improvement. Please refer to the installation process for that or visit lib/karafka/templates for templates of application worker and application controller.
